x-pack/plugins/enterprise_search/public/applications/shared/log_stream/log_stream.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { LogStream } from '../../../../../infra/public';
+
+import { EntSearchLogStream } from './';
+
+describe('EntSearchLogStream', () => {
+  const mockDateNow = jest.spyOn(global.Date, 'now').mockReturnValue(160000000);
+
+  describe('renders with default props', () => {
+    const wrapper = shallow(<EntSearchLogStream />);
+
+    it('renders a LogStream component', () => {
+      expect(wrapper.type()).toEqual(LogStream);
+    });
+
+    it('renders with the enterprise search log source ID', () => {
+      expect(wrapper.prop('sourceId')).toEqual('ent-search-logs');
+    });
+
+    it('renders with a default last-24-hours timestamp if no timestamp is passed', () => {
+      expect(wrapper.prop('startTimestamp')).toEqual(73600000);
+      expect(wrapper.prop('endTimestamp')).toEqual(160000000);
+    });
+  });
+
+  describe('renders custom props', () => {
+    it('overrides the default props', () => {
+      const wrapper = shallow(
+        <EntSearchLogStream sourceId="test" startTimestamp={1} endTimestamp={2} />
+      );
+
+      expect(wrapper.prop('sourceId')).toEqual('test');
+      expect(wrapper.prop('startTimestamp')).toEqual(1);
+      expect(wrapper.prop('endTimestamp')).toEqual(2);
+    });
+
+    it('allows passing a custom hoursAgo that modifies the default start timestamp', () => {
+      const wrapper = shallow(<EntSearchLogStream hoursAgo={1} />);
+
+      expect(wrapper.prop('startTimestamp')).toEqual(156400000);
+      expect(wrapper.prop('endTimestamp')).toEqual(160000000);
+    });
+
+    it('allows passing any prop that the LogStream component takes', () => {
+      const wrapper = shallow(
+        <EntSearchLogStream
+          height={500}
+          highlight="some-log-id"
+          columns={[
+            { type: 'timestamp', header: 'Timestamp' },
+            { type: 'field', field: 'log.level', header: 'Log level', width: 300 },
+          ]}
+          filters={[]}
+        />
```

Comment:
Some more example of queries/columns from another team's usage:

https://github.com/elastic/kibana/blob/1545c6618e5efd6863b2341ecb1b9f798febddfa/x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/TransactionTabs.tsx#L128-L146

Note that the query field uses [KQL](https://www.elastic.co/guide/en/kibana/master/kuery-query.html#kuery-query) which will probably be worth brushing up on, as that's the main way we're going to be using to filter the specific logs a certain view/feature will want.

I'm not 100% about the `filters` prop, will need to dig more into LogStream's typing to see what that takes 🤔